docker-compose.yml

```diff
@@ -3,7 +3,7 @@
 version: "2.1"
 services:
   postgres:
-    image: docker.io/postgres:latest
+    image: ankane/pgvector:latest
```

Comment:
Thank you for unlocking CI. I also observed the same errors on my workstation:

```python
(psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  sorry, too many clients already
```

-- https://github.com/MeltanoLabs/target-postgres/actions/runs/7202377736/job/19620397640?pr=251#step:8:2318

Reply:
It looks like `ankane/pgvector` uses different connection settings/limits? Sigh, always the friendly CI/testing environment... 

Let me know if you endorse this change in its current form, then I can spend some additional cycles on debugging CI ;].

Reply:
Oh I see. The same error also happens when running CI jobs on GH-250. So, it is likely that it's completely unrelated to what I was suspecting about `ankane/pgvector`.